test/MsBuildScript.Tests/runtests.target

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="RunMsBuildScriptTests" 
```

Comment:
The `<?xml` tag, and the `ToolsVersion` and `xmlns` attributes on the Project element aren't necessary.